R/utils-fs.R

```diff
@@ -106,10 +106,7 @@ path_package_pkgdown <- function(package, bs_version = NULL, ...) {
       return(path_package_pkgdown(package, bs_version = NULL, ...))
     }
 
-    stop(
-      package, " does not contain ", src_path("inst/pkgdown/", ...),
-      call. = FALSE
-    )
+    return(character())
```

Comment:
not `NULL` because of the way the template search works: https://github.com/r-lib/pkgdown/blob/3e607f03a54c0edc40e33c299c4dc621e6548f63/R/render.r#L220-L228

I don't want to assume the template package provides templates, as it could provide a configuration file only.

Reply:
Make sense